torch/distributed/distributed_c10d.py

```diff
@@ -2365,7 +2365,7 @@ def batch_isend_irecv(p2p_op_list):
     _check_p2p_op_list(p2p_op_list)
     group = p2p_op_list[0].group
     device = p2p_op_list[0].tensor.device
-    if device.type == "cuda":
+    if device.type == "cuda" or p2p_op_list[0].tensor.is_privateuseone:
```

Comment:
```suggestion
    if device.type == "cuda" or device.type == torch.utils._get_privateuse1_backend_name():
```
Maybe it can work well.

Reply:
Is torch._C._get_privateuse1_backend_name()? I can't find torch.utils._get_privateuse1_backend_name().

Reply:
`torch._C._get_privateuse1_backend_name()` is right